W3ChampionsStatisticService/Program.cs

```diff
@@ -88,13 +88,19 @@
 });
 
 // Configure and add MongoDB
-string mongoConnectionString = Environment.GetEnvironmentVariable("MONGO_CONNECTION_STRING") ?? "mongodb://157.90.1.251:3513"; // "mongodb://localhost:27017";
+string mongoConnectionString = Environment.GetEnvironmentVariable("MONGO_CONNECTION_STRING") ?? "mongodb://157.90.1.251:3513?replicaSet=rs0"; // "mongodb://localhost:27017?replicaSet=rs0";
+if (!mongoConnectionString.Contains("replicaSet="))
```

Comment:
Maybe don't throw an exception for this, just fallback to not using transactions with a warning logged?

Reply:
Supporting two parallel modes is just a headache